.chloggen/issue-44233-parse-invalid-request-in-elbaccess-logs.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. receiver/filelog)
+component: extension/awslogs_encoding
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Parse invalid request in ELB access logs
```

nit - I think originally what we want is to avoid errors for `request_line`. So maybe something like below captures this change better,



```suggestion
note: "Fix ALB log `request_line` parsing for valid formats and avoid errors"
```